Mimir/src/models/PlayerStat.php

```diff
@@ -282,6 +384,7 @@ protected function _getOutcomeSummary(int $playerId, array $rounds)
                             break;
                         } else if ($round->getLoserId() == $playerId) {
                             $acc['feed'] ++;
+                            $acc['points_lost_ron'] -= $this->_getPointsDeltaForRound($round, $playerId, false, $multironRiichiWinners);
```

Comment:
Looks like the `_getPointsDeltaForRound` function is called twice with same arguments and it's quite heavy. I'd suggest memoizing it using some local static cache. Example: https://eddmann.com/posts/implementing-and-using-memoization-in-php/

Reply:
Its calls are in separate code branches, so it is not called twice, but I can move the call before if